ghost/i18n/locales/pa/ghost.json

```diff
@@ -0,0 +1,43 @@
+{
+    "All the best!": "ਸ਼ੁਭਕਾਮਨਾਵਾਂ!",
+    "Complete signup for {{siteTitle}}!": "{{siteTitle}} ਲਈ ਸਾਈਨ-ਅੱਪ ਪੂਰਾ ਕਰੋ!",
+    "Complete your sign up to {{siteTitle}}!": "{{siteTitle}} 'ਤੇ ਆਪਣਾ ਸਾਈਨ-ਅੱਪ ਪੂਰਾ ਕਰੋ!",
+    "Confirm email address": "ਈਮੇਲ ਪਤੇ ਦੀ ਪੁਸ਼ਟੀ ਕਰੋ",
+    "Confirm signup": "ਸਾਈਨ-ਅੱਪ ਦੀ ਪੁਸ਼ਟੀ ਕਰੋ",
+    "Confirm your email address": "ਆਪਣੇ ਈਮੇਲ ਪਤੇ ਦੀ ਪੁਸ਼ਟੀ ਕਰੋ",
+    "Confirm your email update for {{siteTitle}}!": "{{siteTitle}} ਲਈ ਆਪਣੇ ਈਮੇਲ ਅੱਪਡੇਟ ਦੀ ਪੁਸ਼ਟੀ ਕਰੋ!",
+    "Confirm your subscription to {{siteTitle}}": "{{siteTitle}} ਲਈ ਆਪਣੀ ਸਦੱਸਤਾ ਦੀ ਪੁਸ਼ਟੀ ਕਰੋ",
+    "Device:": "ਯੰਤਰ:",
+    "For security verification, enter the code below to sign in to {{siteTitle}}:": "ਸੁਰੱਖਿਆ ਤਸਦੀਕ ਲਈ, {{siteTitle}} ਵਿੱਚ ਸਾਈਨ ਇਨ ਕਰਨ ਲਈ ਹੇਠਾਂ ਦਿੱਤਾ ਕੋਡ ਦਾਖਲ ਕਰੋ:",
+    "For your security, the link will expire in 24 hours time.": "ਤੁਹਾਡੀ ਸੁਰੱਖਿਆ ਲਈ, ਲਿੰਕ 24 ਘੰਟਿਆਂ ਵਿੱਚ ਮਿਆਦ ਪੂਰੀ ਕਰ ਜਾਵੇਗਾ।",
+    "Here's your code to login to {{siteTitle}}": "{{siteTitle}} ਵਿੱਚ ਲੌਗਇਨ ਕਰਨ ਲਈ ਤੁਹਾਡਾ ਕੋਡ ਇਹ ਹੈ",
+    "Hey there,": "ਸਤਿ ਸ੍ਰੀ ਅਕਾਲ,",
+    "Hey there!": "ਸਤਿ ਸ੍ਰੀ ਅਕਾਲ!",
+    "If you did not make this request, you can safely ignore this email.": "ਜੇਕਰ ਤੁਸੀਂ ਇਹ ਬੇਨਤੀ ਨਹੀਂ ਕੀਤੀ ਹੈ, ਤਾਂ ਤੁਸੀਂ ਇਸ ਈਮੇਲ ਨੂੰ ਸੁਰੱਖਿਅਤ ਢੰਗ ਨਾਲ ਨਜ਼ਰਅੰਦਾਜ਼ ਕਰ ਸਕਦੇ ਹੋ।",
+    "If you did not make this request, you can simply delete this message.": "ਜੇਕਰ ਤੁਸੀਂ ਇਹ ਬੇਨਤੀ ਨਹੀਂ ਕੀਤੀ ਹੈ, ਤਾਂ ਤੁਸੀਂ ਇਸ ਸੁਨੇਹੇ ਨੂੰ ਬਸ ਮਿਟਾ ਸਕਦੇ ਹੋ।",
+    "If you didn't try to sign in recently, you can safely ignore this email to deny access.": "ਜੇਕਰ ਤੁਸੀਂ ਹਾਲ ਹੀ ਵਿੱਚ ਸਾਈਨ ਇਨ ਕਰਨ ਦੀ ਕੋਸ਼ਿਸ਼ ਨਹੀਂ ਕੀਤੀ, ਤਾਂ ਪਹੁੰਚ ਤੋਂ ਇਨਕਾਰ ਕਰਨ ਲਈ ਤੁਸੀਂ ਇਸ ਈਮੇਲ ਨੂੰ ਸੁਰੱਖਿਅਤ ਢੰਗ ਨਾਲ ਨਜ਼ਰਅੰਦਾਜ਼ ਕਰ ਸਕਦੇ ਹੋ।",
+    "Please confirm your email address with this link:": "ਕਿਰਪਾ ਕਰਕੇ ਇਸ ਲਿੰਕ ਨਾਲ ਆਪਣੇ ਈਮੇਲ ਪਤੇ ਦੀ ਪੁਸ਼ਟੀ ਕਰੋ:",
+    "Secure sign in link for {{siteTitle}}": "{{siteTitle}} ਲਈ ਸੁਰੱਖਿਅਤ ਸਾਈਨ-ਇਨ ਲਿੰਕ",
+    "See you soon!": "ਜਲਦੀ ਮਿਲਦੇ ਹਾਂ!",
+    "Sent to {{email}}": "{{email}} ਨੂੰ ਭੇਜਿਆ ਗਿਆ",
+    "Sign in": "ਸਾਈਨ ਇਨ ਕਰੋ",
+    "Sign in to {{siteTitle}}": "{{siteTitle}} 'ਤੇ ਸਾਈਨ ਇਨ ਕਰੋ",
+    "Sign in verification": "ਸਾਈਨ ਇਨ ਤਸਦੀਕ ਕਰੋ",
```

Comment:
This is a header for where a user can type in their code, not directions.  AI recommends  "ਸਾਈਨ ਇਨ ਤਸਦੀਕ" or "ਸਾਈਨ ਇਨ ਦੀ ਤਸਦੀਕ" instead.